src/App.tsx

```diff
@@ -96,8 +96,8 @@ const navigation: NavigationItems = [
     resourceName: tables.incident
   },
   {
-    name: "Config",
-    href: "/configs",
+    name: "Catalog",
+    href: "/catalogs",
```

Comment:
I think lets use catalog instead og catalogs - there is 1 single unified catalog